lib/sunrise/sunrise.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Automattic\VIP\Sunrise;
+
+/**
+ * Nothing to see here for single sites
+ */
+if ( ! is_multisite() ) {
+	return;
+}
+
+/**
+ * Log errors retrieving network for a given request
+ *
+ * @param string $domain
+ * @param string $path
+ */
+function network_not_found( $domain, $path ) {
+	$data = [
+		'domain_requested' => $domain,
+		'path_requested'   => $path,
+	];
+	$data = json_encode( $data );
+
+	error_log( 'Network Not Found! ' . $data );
+}
+add_action( 'ms_network_not_found', __NAMESPACE__ . '\network_not_found', 9, 2 );
```

Comment:
Why do we need priority `9`? Should probably have an inline comment with the reason.